react/tic-tac-toe/src/app/index.js

```diff
@@ -1,11 +1,32 @@
-import React from 'react';
+import React, { Component } from 'react';
 
-import Game from './screens/Game';
+import MatchesService from '../services/MatchesService';
 
+import Game from './screens/Game';
+import Matches from './screens/Matches';
 import '../scss/application.scss';
+import styles from './styles.module.scss';
+
+class App extends Component {
+  state = {
+    data: []
+  }
+
+  componentDidMount() {
```

Comment:
As said before, this method should be in the `Matches` component scope.
Then `App` could go back to being a function again.